helm_deploy/values-prod.yaml

```diff
@@ -17,6 +17,7 @@ generic-service:
     COMMUNITY_ACCOMMODATION_API_SERVICE_NAME: approved-premises
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
+    NEW_WITHDRAWALS_FLOW_DISABLED: 1
```

Comment:
I might consider putting this into every environment so your e2e tests will run against it and any manual QA you do will have high parity

Reply:
Yeah, I considered that, but that means we'd have to change the e2e tests